src/providers/Project/projectHooks.ts

```diff
@@ -22,7 +22,13 @@ function cloneProject(client: any, project: Project) {
   const localProject = createLocalProject(
     project.id,
     project.seed,
-    project.accounts.map(acc => acc.draftCode),
+    project.accounts.map(acc => acc.state),
+
+    project.contracts.map(con => ({
+      code: con.script,
```

Comment:
Back to that `code` vs `script` name :)

Reply:
yes, i will refactor that

Reply:
this commit addressed this: [300f2c6ce469f16f009cc8eda72d105a7fad8f8f](https://github.com/soetop/flow-playground/commit/300f2c6ce469f16f009cc8eda72d105a7fad8f8f)